sdv/constraints/tabular.py

```diff
@@ -293,31 +327,31 @@ def _get_is_datetime(self, table_data):
 
         return is_datetime
 
+    def _check_columns_exist(self, table_data, field):
+        values = getattr(self, f'_{field}')
+        exists = [column not in table_data.columns for column in values]
```

Comment:
I would rename `exists` to `missing` (otherwise the `if any(exists)` is very confusing)
This can also be rewritten using sets:

```python3
if set(values) - set(table_data.columns):
    raise ...
```